package.json

```diff
@@ -33,18 +33,18 @@
   "dependencies": {
     "@shagstrom/split-pane": "0.9.4",
     "celldown.js": "0.1.0",
-    "chokidar": "1.6.1",
+    "chokidar": "2.0.0",
```

Comment:
New major versions generally introduce breaking changes. Therefore we can't move to chokidar 2 unless we make sure this is actually working (and fix things if this doesn't). This PR need further investigation IMO.

Reply:
I agree. For now, should you just merge the PR with all the other fixes?

Reply:
:+1: Feel free to update your PR (and please be careful of merge conflicts).